arrow/parser.py

```diff
@@ -582,7 +753,13 @@ def parse(cls, tzinfo_string):
             iso_match = cls._TZINFO_RE.match(tzinfo_string)
 
             if iso_match:
-                sign, hours, minutes = iso_match.groups()
+                (
+                    sign,
+                    hours,
+                    minutes,
+                ) = (
+                    iso_match.groups()
+                )  # type: Optional[str], str, Union[str, int, None]
```

Comment:
Should we declare `sign`, `hours`, and `minutes` types before to avoid the commented types?

Reply:
Done!